plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -118,6 +119,16 @@ class ScaleBarImpl : ScaleBar, View {
       // Refresh mapViewWidth
       mapViewWidth = mapViewWidth
       field = value
+      (layoutParams as FrameLayout.LayoutParams).apply {
+        gravity = value.position
+        setMargins(
+          value.marginLeft.toInt(),
```

Comment:
should we support `marginStart` and `marginEnd` instead of `marginLeft` and `marginRight`?